### PR TITLE
Microverse first capestone 

### PR DESCRIPTION
**Pull request by Baroka-wp**
- [ ] Milestone 1 : Choose the content
 - [ ] Milestone 2 : Build the 2 pages for desktop version
 - [ ] Milestone 3: Build the 2 pages for mobile version
 - [ ] Milestone 4 : Make dynamic feature with JavaScript (speaker feature, popup, components for Header and footer)
 - [ ] Milestone 5 : Documentation and loom video for presentation

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -319,42 +431,55 @@ section.partners {
...
-      justify-content: center;
-      align-items: center;
-      gap: 5px;
-      width: 30%; }
-    footer .footer-content .copyrigth {
-      width: 70%; }
+  background-color: #d3d3d3;
+}
+
+footer .footer-content {
```
Comment (omar-muhamad) [inline style/style.css:458]: - [ ] Please make the background image on the index page in a dark color like the given.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Baroka-wp,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔
- your code is well structured and clean. ✔

## (Changes Required) Aspects to improve: ♻️

- You did a great job so far but you have some design requirements to improve:

- [ ] For the headline section on the index page in the desktop view, it is supposed to be centered in the middle so0 please fix that.
- [ ] for the same section in the mobile view it is supposed to be in the center so please fix that and the box which has the` <p>` tag with description suppose to have more width like the given design so kindly fix that.

| Given Design  | Your Design |
| ------------- | ------------- |
| image | image |
| image  | image  |

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Baroka-wp** [comment]
Hello @Omar-Muhamad, thank you for your review. And I appreciate your hard work🥇  
I already make the Change required about hero centered and footer about in black. I request another review.

**Comment by omar-muhamad** [comment]
> Hello @Omar-Muhamad, thank you for your review. And I appreciate your hard work🥇  
> I already make the Change required about hero centered and footer about in black. I request another review. 
> 

I saw your changes everything seems good now.
Have a great day

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Baroka-wp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Baroka-wp** [comment]
@Omar-Muhamad  thank you very much, 
Have a great day !

**Pull request merged**